dags/utils/acoustic/acoustic_client.py

```diff
@@ -152,7 +152,8 @@ def generate_report(
 
         supported_report_types = ("raw_recipient_export", "contact_export",)
         if report_type not in supported_report_types:
-            pass  # TODO: raise with options
+            err_msg = f"{report_type} is not a valid option, support report types: {supported_report_types}"
+            raise AttributeError(err_msg)
 
         with open(request_template, "r") as _file:
```

Comment:
Ideally we could use more explicit names for variables, so that if they are used later in the code is easier to know what they refer to. In this case for example: _template_file or _request_file better than _file.

Reply:
This is just file object being loaded into memory and used only on the next line.